pkg/domain/nexpose.go

```diff
@@ -21,11 +21,3 @@ type NexposeVulnerability struct {
 	Description    string
 	Title          string
 }
-
-// NexposeClient represents an interface used to interact with the Nexpose API
```

Comment:
I'm not able to comment on lines that aren't a part of the diff so leaving the comment here.

task: It appears that `NexposeVulnerability` and `NexposeAssetVulnerability` are no longer used to communicate across packages. If that's correct then they should be moved to the package where they are used.